js/modules/keyboard-navigation.src.js

```diff
@@ -1117,6 +1102,31 @@ H.Chart.prototype.addKeyboardNavigationModules = function () {
             }
         }),
 
+        // Reset zoom
+        navModuleFactory('resetZoom', [
+            // Tab/Up/Down/Left/Right - just move
+            [[9, 37, 38, 39, 40], function (keyCode, e) {
```

Comment:
Will suggest using a map for improving readability.
E.g:
```javascript
var keyCodes: {
  tab: 9,
  up: 37.
  // ...
};
Object.keys(keyCodes).forEach(keyCode, e) {
  // ...
    keyCode === keyCodes.tab && e.shiftKey ||
  // ...
});
// ....
```

Reply:
Agree, but that is a bigger refactoring and should be put off to a later commit when we do the options refactoring etc for 7.1. Note that the above is not a forEach, but an array of numbers that get the same handler function in the navModule, so it is not a straight refactor.